tests/lib/prepare-project.sh

```diff
@@ -32,6 +44,28 @@ build_deb(){
     cp ../*.deb "$GOHOME"
 }
 
+fedora_build_rpm() {
```

Comment:
I think we can call this as build_rpm and in the future just have the deltas internally, instead of having completely different instructions for each distro.